specification/edgeorder/resource-manager/Microsoft.EdgeOrder/2020-12-01-preview/edgeorder.json

```diff
@@ -0,0 +1,3133 @@
+          },
+          {
+            "name": "cancellationReason",
+            "in": "body",
+            "description": "Reason for cancellation.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CancellationReason"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "204": {
+            "description": "\"No content\""
+          },
+          "default": {
+            "description": "Error response describing reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CancelOrder": {
+            "$ref": "./examples/OrderCancel.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EdgeOrder/orders/{orderName}/return": {
+      "post": {
+        "tags": [
+          "Orders"
+        ],
+        "description": "Return order.",
+        "operationId": "ReturnOrder",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/orderNameParameter"
+          },
+          {
+            "$ref": "../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "returnOrderDetails",
+            "in": "body",
+            "description": "Return order CurrentStatus.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReturnOrderDetails"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "202": {
+            "description": "\"Return order request accepted.\""
+          },
+          "default": {
+            "description": "Error response describing reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "ReturnOrder": {
+            "$ref": "./examples/OrderReturn.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AdditionalErrorInfo": {
+      "type": "object",
+      "properties": {
+        "type": {
```

Comment:
Please mark the properties as read-only.

Reply:
We are referring AdditionalErrorInfo from externall library we cannot add read-only